src/graphmap.rs

```diff
@@ -59,7 +59,7 @@ pub type DiGraphMap<N, E> = GraphMap<N, E, Directed>;
 /// Depends on crate feature `graphmap` (default).
 #[derive(Clone)]
 pub struct GraphMap<N, E, Ty> {
-    nodes: IndexMap<N, Vec<(N, CompactDirection)>>,
+    pub(crate) nodes: IndexMap<N, Vec<(N, CompactDirection)>>,
```

Comment:
If these `pub(crate)` additions are not wanted, the `par_nodes` definition could be moved into this module or a submodule.

Reply:
Yes, I say move `par_nodes` to here instead.

Reply:
Done. I've removed the `rayon` module. Everything has been put into `graphmap.rs`, which on reflection, is quite a bit nicer than what I first proposed.